Index: common/build.config.common
===================================================================
--- common/build.config.common	2025-10-14 12:24:31.050395112 +1030
+++ common/build.config.common	2025-10-14 12:24:31.046454136 +1030
@@ -14,7 +14,7 @@
     echo "${ROOT_DIR}/${BUILDTOOLS_PREBUILT_BIN}/dtc";
 })
 
-KCFLAGS="${KCFLAGS} -D__ANDROID_COMMON_KERNEL__"
+KCFLAGS="${KCFLAGS} -march=armv8.7-a+crc+crypto+lse+bf16+i8mm+dotprod+rcpc+pauth+bti -mtune=generic -D__ANDROID_COMMON_KERNEL__"
 EXTRA_CMDS=''
 STOP_SHIP_TRACEPRINTK=1
 IN_KERNEL_MODULES=1
