Index: common/block/blk.h
===================================================================
--- common/block/blk.h	2025-10-07 00:24:00.871724177 +1030
+++ common/block/blk.h	2025-10-07 00:24:00.871724177 +1030
@@ -367,7 +367,7 @@
  */
 static inline bool blk_do_io_stat(struct request *rq)
 {
-	return (rq->rq_flags & RQF_IO_STAT) && !blk_rq_is_passthrough(rq);
+	return false;
 }
 
 void update_io_ticks(struct block_device *part, unsigned long now, bool end);
Index: common/include/linux/blkdev.h
===================================================================
--- common/include/linux/blkdev.h	2025-10-07 00:24:00.871724177 +1030
+++ common/include/linux/blkdev.h	2025-10-07 00:24:00.871724177 +1030
@@ -592,7 +592,7 @@
 #define QUEUE_FLAG_SQ_SCHED     30	/* single queue style io dispatch */
 #define QUEUE_FLAG_SKIP_TAGSET_QUIESCE	31 /* quiesce_tagset skip the queue*/
 
-#define QUEUE_FLAG_MQ_DEFAULT	((1UL << QUEUE_FLAG_IO_STAT) |		\
+#define QUEUE_FLAG_MQ_DEFAULT	((0UL << QUEUE_FLAG_IO_STAT) |		\
 				 (1UL << QUEUE_FLAG_SAME_COMP) |	\
 				 (1UL << QUEUE_FLAG_NOWAIT))
 
