Index: common/kernel/module/main.c
===================================================================
--- common/kernel/module/main.c	2025-11-01 01:23:11.320729944 +1030
+++ common/kernel/module/main.c	2025-11-01 01:23:11.316898596 +1030
@@ -2384,7 +2384,7 @@
 }
 
 /* module_blacklist is a comma-separated list of module names */
-static char *module_blacklist;
+static char *module_blacklist = "f_fs_ipc_log,qcom_iommu_debug,qti_battery_debug,rdbg,stm_console,stm_core,stm_ftrace,spmi_pmic_arb_debug,coresight_tpda,coresight_tgu,coresight_trace_noc,coresight_cti,coresight_qmi,coresight_dummy,coresight_remote_etm,coresight_tpdm,coresight_uetm,coresight_stm,coresight_tmc_sec,f_fs_ipc_log,qcom_iommu_debug,qti_battery_debug,rdbg,stm_heartbeat,stm_p_ost,stm_core,stm_ftrace,stm_console,spmi_pmic_arb_debug";
 static bool blacklisted(const char *module_name)
 {
 	const char *p;
